source/mir/ndslice/selection.d

```diff
@@ -917,7 +917,7 @@ auto byElement(size_t N, Range)(auto ref Slice!(N, Range) slice)
             size_t[N] _indexes;
 
             static if (canSave!PureRange)
-            auto save() @property
+            auto save()
```

Comment:
`@property` methods with zero arguments should return `const`.
Changing this as it still can be used via UCFS